config/config.go

```diff
@@ -33,15 +39,32 @@ type IngressConfig struct {
 func Get() *IngressConfig {
 
 	options := viper.New()
-	options.SetDefault("MaxSize", 10*1024*1024)
-	options.SetDefault("Port", 3000)
-	options.SetDefault("StageBucket", "available")
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
+		cfg := clowder.LoadedConfig
+		topic, ok := clowder.KafkaTopics["platform.upload.advisor"]
```

Comment:
Is this the topic that should be created by ingress? It writes to more than just the advisor topic so I'm just trying to understand what this bit does.

Reply:
It will be. That's generated in the clowdapp. What topics does ingress need? Kyle suggested adding `platform.payload-status`

Reply:
ingress doens't consume anything, but it produces to all the topics in the "VALID_TOPICS" environment variable.